rotkehlchen/rotkehlchen.py

```diff
@@ -434,6 +435,13 @@ def deactivate_premium_status(self) -> None:
         self.accountant.deactivate_premium_status()
         self.chain_manager.deactivate_premium_status()
 
+    def activate_premium_status(self, credentials: PremiumCredentials) -> None:
```

Comment:
you don't need to pass the credentials here. The only place you use this is in the task manager right after `premium_create_and_verify` has succeeded. When that succeeds it returns a `Premium` object. Take that object and pass it to this method and then skip the `Premium` initialization in this method.